client/ayon_core/plugins/publish/extract_thumbnail.py

```diff
@@ -517,9 +530,45 @@ def _create_frame_from_video(self, video_file_path, output_dir):
             # run subprocess
             self.log.debug("Executing: {}".format(" ".join(cmd)))
             run_subprocess(cmd, logger=self.log)
-            self.log.debug(
-                "Thumbnail created: {}".format(output_thumb_file_path))
-            return output_thumb_file_path
+
```

Comment:
If ffmpeg can produce empty output image we should make sure the file is removed. I can't add suggestion to unchanged lines, so I'll try to visualize how it might look.

```python
try:
    # run subprocess
    ...

except RuntimeError as exc:
    ...

finally:
    # Remove output file if is empty
    if (
        os.path.exists(output_thumb_file_path)
        and os.path.getsize(output_thumb_file_path) == 0
    ):
        os.remove(output_thumb_file_path)

Reply:
done in [9791fda](https://github.com/ynput/ayon-core/pull/1250/commits/9791fda4f6a4e8b898227700f9df4a119845c97b)